smart-contracts/contracts/BridgeBank/BridgeBank.sol

```diff
@@ -67,20 +67,27 @@ contract BridgeBank is BankStorage, CosmosBank, EthereumWhiteList, CosmosWhiteLi
    */
   bool private _reinitialized;
 
+  /**
+   * @dev The address of the Rowan Token
+   */
+   address public rowanTokenAddress;
+
   /**
    * @notice Initializer
    * @param _operator Manages the contract
    * @param _cosmosBridgeAddress The CosmosBridge contract's address
    * @param _owner Manages whitelists
    * @param _pauser Can pause the system
    * @param _networkDescriptor Indentifies the connected network
+   * @param _rowanTokenAddress The address of the Rowan ERC20 contract on this network
    */
   function initialize(
     address _operator,
     address _cosmosBridgeAddress,
     address _owner,
     address _pauser,
-    int32 _networkDescriptor
+    int32 _networkDescriptor,
+    address _rowanTokenAddress
```

Comment:
Why are we adding a parameter to initialize?  Can't ever be called in production.

Reply:
This contract will be deployed on other networks, and while the address will almost certainly be the zero address on other chains, it should be something that is set on deployment. Since this is an open-source project, it makes sense to me that if someone wanted to deploy these contracts, the initialized code would still be updated with our changes.